docker/dockerdist/dockerdist.go

```diff
@@ -36,8 +36,8 @@ import (
 	"github.com/docker/docker/reference"
 	"github.com/docker/docker/registry"
 	"github.com/opencontainers/go-digest"
-	"github.com/spf13/viper"
 	"golang.org/x/net/context"
+	"github.com/spf13/viper"
```

Comment:
Ive never heard of this (its my first time writing Go code...)
It looks cool though, Ill run this